docs/commands/bundle.md

```diff
@@ -1,117 +1,91 @@
-  --max-problems       Reduce output to max N problems.  [number] [default: 100]
-  --ext                Bundle file extension.   [choices: "json", "yaml", "yml"]
-  --skip-rule          Ignore certain rules.                             [array]
-  --skip-preprocessor  Ignore certain preprocessors.                     [array]
-  --skip-decorator     Ignore certain decorators.                        [array]
-  --dereferenced, -d   Produce fully dereferenced bundle.              [boolean]
-  --force, -f          Produce bundle output even when errors occur.   [boolean]
-  --config             Specify path to the config file.                 [string]
-```
+:::success Tip
+To learn more about preprocessors, rules, and decorators, please refer to [this](../custom-rules.md) document.
+:::
 
-The command:
+## Usage
 
 ```bash
-openapi bundle --output <outputName> --ext <ext> [entrypoints...]
-```
-
-- `[entrypoints...]` corresponds to the name(s) of your root document(s).
-- Instead of full paths, you can use aliases assigned in your `apiDefinitions` within your `.redocly.yaml` configuration file as entrypoints.
-- The `<outputName>` is your desired output filename or folder. If the folder doesn't exist, it's automatically created.
-- Use `--ext <ext>` to specify the type and extension of the output file: `.json`, `.yml` or `.yaml`.
+openapi bundle <entrypoints>...
+openapi bundle <entrypoints> [--max-problems=<n>]
+openapi bundle <entrypoints> [--lint] [--config=<path>]
+openapi bundle <entrypoints>... -o <outputName> --ext <ext>
+openapi bundle --version
+```
+
+## Options
+
+option                 | type      | required?    | default     | description
+-----------------------|:---------:|:------------:|:-----------:|------------
+`entrypoints`          | `array`   | yes          | `[]`        | Array of API root definition filenames that need to be bundled. Instead of full paths, you can use aliases assigned in your `apiDefinitions` within your `.redocly.yaml` configuration file as entrypoints.
+`--config`             | `string`  | no           | -           | Specify path to the config file
+`--dereferenced`, `-d` | `boolean` | no           | -           | Generate fully dereferenced bundle
+`--ext`                | `string`  | no           | `yaml`      | Specify bundled file extension.<br />**Possible values:** `json`, `yaml`, `yml`
+`--force`, `-f`        | `boolean` | no           | -           | Generate bundle output even when errors occur
+`format`               | `string`  | no           | `codeframe` | Format for the output.<br />**Possible values:** `codeframe`, `stylish`
+`--help`               | `boolean` | no           | -           | Show help
+`--lint`               | `boolean` | no           | `false`     | Lint definition files
+`--max-problems`       | `number`  | no           | 100         | Truncate output to display the specified maximum number of problems
```

Comment:
question: What's the allowed range of values here (min to max)?

Reply:
If we don't have a conclusive answer to this, I think you can just leave the text as it is, without extra details about min-max. In that case, we should open a GitHub issue to investigate and document this at some later point, outside of the GSOD project. Does that sound OK?

Reply:
Yup, totally OK. I will create an issue with undocumented/need clarification parts. However, I hope to receive some clarifications from developers during the GSOD project :)